docs/resources/connection.md

```diff
@@ -146,7 +158,9 @@ The following arguments are supported:
 
 - `handle` - (Required) A friendly identifier for your connection, and must be unique across your connections.
 - `plugin` - (Required) The name of the plugin.
-- `config` - (Optional) Configuration for the connection.
+- `config` - (Optional) JSON Configuration for the connection, stored in state (cannot be used with `config_wo`). Note: As secrets are not returned from the API, this may show perpetual config drift is secrets are included in this argument.
+- `config_wo` - (Optional) Write-Only JSON Configuration for the connection, **NOT** stored in state  (cannot be used with `config`). Requires indication of changes using `config_wo_version`.
```

Comment:
```suggestion
- `config_wo` - (Optional) Write-Only JSON configuration for the connection, **NOT** stored in state  (cannot be used with `config`). Requires indication of changes using `config_wo_version`.
```

Reply:
@graza-io check the casing of `Write-Only` against the TF docs to be consistent - is it `Write-only`?